src/routes/wallet/wallet.routes.js

```diff
@@ -465,4 +466,45 @@ walletRouter.post(
  */
 walletRouter.post('/stop', stop);
 
+/**
+ * PUT request to mark inputs from a txHex as used.
+ * For the docs, see api-docs.js
+ */
+walletRouter.put(
+  '/utxos-selected-as-input',
```

Comment:
In the `atomic-swap` routes we have a simular router, `/unlock`, which does the same for an atomic swap `PartialTx`. I was thinking about this and if it would be better to unify both APIs. 

What do you think?

Reply:
I think that unifying these APIs would be the best choice, but I think we should do this on another PR.
Because deprecating the unlock API will require changing the docs and the website tutorial, maybe we should organize this on another issue.

What do you think?

Reply:
I agree. Can you create the issue for this, please?